src/objects/DataImport__c.object

```diff
@@ -832,6 +832,18 @@
             </valueSetDefinition>
         </valueSet>
     </fields>
+    <fields>
+        <fullName>Donation_Elevate_Recurring_ID__c</fullName>
+        <description>The external, unique ID of the Recurring Donation in Elevate that this donation Opportunity is linked to.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The external, unique ID of the Recurring Donation in Elevate that this donation Opportunity is linked to.</inlineHelpText>
+        <label>Donation Elevate Recurring ID</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
```

Comment:
@callen-sfdo Is this meant to be marked as unique and as External Id?

Reply:
@davidmreed i'm not making it unique on either because we will definitely have duplicates on the opportunity, and may need to allow them on the DI.  I will be making the one on opportunity an external id since it will be used in trigger queries.